desktop/apps/artist/stylesheets/sections.styl

```diff
@@ -34,6 +34,9 @@
   flex-direction row
   &__text
     margin-left 13px
+    &__show-name
```

Comment:
I haven't looked into stylus for a while and it changed quite a bit. The [partial reference](http://stylus-lang.com/docs/selectors.html#partial-reference) thing might went a bit too far. 😛